lib/offline/storage.js

```diff
@@ -1067,13 +1081,38 @@ shaka.offline.Storage = class {
           downloadGroup,
           request,
           estimator.getInitSegmentEstimate(stream.id),
+          true,
```

Comment:
Since boolean literals are hard to interpret without a hint, please use an inline comment to label the argument.  For example: `/* isInitSegment */ true,`

Reply:
Done!